docs/docs/user-guides/client-instructions.mdx

```diff
@@ -3,7 +3,9 @@ title: Client Instructions
 sidebar_position: 4
 ---
 
-## Install and Setup
+# Client instructions (for Firezone users)
+
+## Install and setup
```

Comment:
I generally try to split up multiple titles with descriptions or text in between so you don't end up with:

# Title 1
## Heading 2
### Heading 3
Actual content

```suggestion
# End-user Client Instructions

Follow these instructions if you're an end-user trying to set up your WireGuard client to work with Firezone.

## Install and setup
```